extract/extract/extractors/zalando.py

```diff
@@ -155,21 +151,20 @@ def __get_sustainability_info(
             title.string: description.string for title, description in zip(titles, descriptions)
         }
 
-    else:
-        return {}
+    return {}
```

Comment:
See above. Don't you think with `else` it's more explicit what happens?

Reply:
If you have kept this style in the project than its probably better to keep it like this. I was running pylint over the code and it suggested to remove it, see here: https://pheanex.github.io/pylint/

Reply:
Ok, agreed!